Sources/SolanaSwift/APIClient/APIClient.swift

```diff
@@ -381,14 +389,37 @@ extension SolanaAPIClient {
     }
     
     public func setLogFilter(filter: String) async throws -> String? {
-        let result: String? = try await self.get(method: "setLogFilter", params: [filter])
-        return result
+        try await self.get(method: "setLogFilter", params: [filter])
     }
     
     public func validatorExit() async throws -> Bool {
         try await self.get(method: "validatorExit", params: [])
     }
     
+    public func getMultipleMintDatas(mintAddresses: [String], programId: String = PublicKey.tokenProgramId.base58EncodedString) async throws -> [String: Mint] {
```

Comment:
I think we should keep APIClient much simple and identical to official docs as possible. TokenRepository also share the domain of issue, so it be better if we move them to `TokenRepository/APIClient+Extension.swift`

Reply:
Sounds good